.gitignore

```diff
@@ -33,11 +33,8 @@ Vagrantfile
 /certs/saml.crt
 /config/application.yml
 /config/aws.yml
-/config/database.yml
 /config/newrelic.yml
-/config/saml.key.enc
```

Comment:
What happened here is that you probably had lock untracked files by this name, that were previously hidden by .gititnore.  The correct fix is probably to locally remove them.